### PR TITLE
RBAC known issue

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/2312#issuecomment-630414399

### CODE REVIEW

File: release_notes/known_issues.md [hunk 1]
```diff
@@ -170,9 +170,9 @@ You can create and apply multiple certificate policies on a single managed clust
 
 For more information, see [Certificate policy controller](../security/cert_policy_ctrl.md).
 
+### Any user can create and import clusters
 
-
-
+In OpenShift Container Platform, any authenticated user can provision projects and have administrator privileges. Any user can create and import clusters from Red Hat Advanced Cluster Management for Kubernetes. As a result, some roles have incorrect access privileges for the hub cluster. In order to create a cluster and generate commands, you must have cluster administrator privileges. For more information view the [Role based access control (RBAC) table](../security/security_intro.md).
```
Comment (ckandag) [inline release_notes/known_issues.md:176]: In OpenShift Container Platform, any authenticated user can provision projects and have administrator privileges to the project and associated namespace.  As the administrator of a namespace, a user can generate commands to import clusters into Red Hat Advanced Cluster Management for Kubernetes Hub Cluster.  However,  user would need to have cluster administrator privileges on the Managed Cluster in order to execute these commands to import.

File: release_notes/known_issues.md [hunk 2]
```diff
@@ -170,9 +170,9 @@ You can create and apply multiple certificate policies on a single managed clust
 
 For more information, see [Certificate policy controller](../security/cert_policy_ctrl.md).
 
+### Any user can create and import clusters
```
Comment (ckandag) [inline release_notes/known_issues.md:173]: Any authenticated user can import clusters

File: release_notes/known_issues.md [hunk 3]
```diff
@@ -170,9 +170,9 @@ You can create and apply multiple certificate policies on a single managed clust
 
 For more information, see [Certificate policy controller](../security/cert_policy_ctrl.md).
 
-### Any user can create and import clusters
+### Any authenticated user can import clusters
 
-In OpenShift Container Platform, any authenticated user can provision projects and have administrator privileges. Any user can create and import clusters from Red Hat Advanced Cluster Management for Kubernetes. As a result, some roles have incorrect access privileges for the hub cluster. In order to create a cluster and generate commands, you must have cluster administrator privileges. For more information view the [Role based access control (RBAC) table](../security/security_intro.md).
+In OpenShift Container Platform, any authenticated user can provision projects with administrator privileges to the project and associated namespace. As the administrator of a namespace, users can import clusters into Red Hat Advanced Cluster Management for Kubernetes. In order to create a cluster and generate commands on the hub cluster, you must have cluster administrator privileges. For more information view the [Role based access control (RBAC) table](../security/security_intro.md).
```
Comment (ckandag) [inline release_notes/known_issues.md:175]: As the administrator of a namespace, users can **"generate commands"** to import clusters into Red Hat Advanced Cluster Management for Kubernetes.

However , inorder to execute the cmds to import cluster, user would need cluster-admin privileges on the **"managed cluster"**

**Comment by dockerymick** [comment]
@cadawson I addressed the comments and suggestions made by @ckandag. I am go to go ahed and merge this.

File: release_notes/known_issues.md [hunk 4]
```diff
@@ -170,9 +170,9 @@ You can create and apply multiple certificate policies on a single managed clust
 
 For more information, see [Certificate policy controller](../security/cert_policy_ctrl.md).
 
+### Any authenticated user can import clusters
 
-
-
+In OpenShift Container Platform, any authenticated user can provision projects with administrator privileges to the project and associated namespace. As the administrator of a namespace, users can import clusters into Red Hat Advanced Cluster Management for Kubernetes. In order to run the generated import commands on the managed cluster, you must have cluster administrator privileges. For more information view the [Role based access control (RBAC) table](../security/security_intro.md).
```
Comment (cadawson) [inline release_notes/known_issues.md:175]: Any authenticated user of OpenShift Container Platform can provision projects and have administrator privileges to the project and its associated namespace. As the administrator of a namespace, you can generate commands to import clusters that you want to manage with the Red Hat Advanced Cluster Management for Kubernetes hub cluster. To run the generated commands and import the cluster, you must have cluster administrator privileges on the managed cluster.

**Pull request merged**